### ISSUE TITLE
Update for new API

### ISSUE THREAD

**Issue by ghenry** [bigpresh/SMS-AQL#3]
Hi,

This has stopped working tonight. Can you check the code?

Thanks.

**Comment by bigpresh** [comment]
From a quick test, it worked for me:

```
[davidp@supernova:~/dev/git/SMS-AQL]$ perl eg/example.pl 
SMS::AQL 1.02 loaded OK
Account sms-aql-test has 212 credits.
Destination: ?> +447<snipped>
Message: ?> Test from Dave
Status: Successful,  Reason: OK, Server response: AQSMS-OK:1
```

What's happening for you?  What error message do you get?

From a quick look on AQL's site, I do see that they have a new API ... and their [old API documentation](https://aql.com/wp-content/uploads/2020/08/Send-SMS-API.pdf) says:
> This SMS API is deprecated. This has been superseded by the aql RESTful API.
Documentation can be found at https://api.aql.com/doc/

For my future reference, [AQL's new RESTful API documentation](https://api.aql.com/doc/) looks well designed and documented, so updating SMS::AQL to use it shouldn't be too difficult.

**Comment by ghenry** [comment]
Yeah, it came back :-) I really need to do a PR for this. Thanks for
replying!
